src/PointwiseFunctions/SpecialRelativity/LorentzBoostMatrix.hpp

```diff
@@ -53,5 +56,51 @@ template <size_t SpatialDim>
 void lorentz_boost_matrix(
     gsl::not_null<tnsr::Ab<double, SpatialDim, Frame::NoFrame>*> boost_matrix,
     const tnsr::I<double, SpatialDim, Frame::NoFrame>& velocity);
+
+template <size_t SpatialDim>
+tnsr::Ab<double, SpatialDim, Frame::NoFrame> lorentz_boost_matrix(
+    const std::array<double, SpatialDim>& velocity);
+/// @}
+
+/// @{
+/*!
+ * \ingroup SpecialRelativityGroup
+ * \brief Apply a Lorentz boost to the spatial part of a one form.
```

Comment:
One form. This is because here and in SpEC they chose (rather confusingly) the lorentz boost matrix to be the usual _inverse_ lorentz boost matrix (or that for one forms).

Reply:
The argument type is a vector (`tnsr::I`), so I also think this documentation should read `vector`. Refer to the docs of `lorentz_boost_matrix` above for details and math. I also find those docs confusing though. I believe they say that $\Lambda^a_\bar{a}$ is computed, which boosts a vector from barred to unbarred or a one-form from unbarred to barred (consistent with Blandford/Thorne Sec. 2.7). So my suggestion is:

- The docs and argument for this function should read `vector` (they boost a vector)
- The function for the one-form below should apply the inverse Lorentz transformation (by inverting the sign of the velocity). This means you add a minus sign to the function below and remove the minus signs from where the function is used in Kerr-Schild. Make sure to clarify this in the docs of the function.

@geoffrey4444 can you confirm or clarify this?

Reply:
You are right Nils. My mistake here, Niko. It is the transformation for a vector. But I should take the inverse Lorentz matrix (or add a sign to the velocity) as they transform with the inverse matrix wrt one forms.

Reply:
I think the sign for this function is correct, but the one below (for one-forms) should have a minus sign.